src/ILCompiler.ReadyToRun/src/ILCompiler.ReadyToRun.csproj

```diff
@@ -29,7 +29,13 @@
     <Compile Include="CodeGen\ReadyToRunObjectWriter.cs" />
     <Compile Include="Compiler\DependencyAnalysis\CompilerIdentifierNode.cs" />
     <Compile Include="Compiler\DependencyAnalysis\ReadyToRunCodegenNodeFactory.cs" />
-    <Compile Include="Compiler\DependencyAnalysis\CoreCLRReadyToRunHeaderNode.cs" />
+    <Compile Include="Compiler\DependencyAnalysis\HeaderNode.cs" />
```

Comment:
Nit: The convention is to match the namespace and directory.

Reply:
Fixed in 7th commit, thanks for pointing that out!